runner/dl_checkpoints.sh

```diff
@@ -79,7 +79,42 @@ function download_all_models() {
 
     # Download live-video-to-video models.
     huggingface-cli download KBlueLeaf/kohaku-v2.1 --include "*.safetensors" "*.json" "*.txt" --exclude ".onnx" ".onnx_data" --cache-dir models
-    huggingface-cli download KwaiVGI/LivePortrait --include "*.safetensors" "*.json" "*.txt" --exclude ".onnx" ".onnx_data" --cache-dir models
+    huggingface-cli download stabilityai/sd-turbo --include "*.safetensors" "*.json" "*.txt" --exclude ".onnx" ".onnx_data" --cache-dir models
+    huggingface-cli download warmshao/FasterLivePortrait --local-dir models/FasterLivePortrait--checkpoints
+}
+
+function build_tensorrt_models() {
```

Comment:
I wonder if we want to have one function to download and build all models. Is that what Orchestrators do, do they download ALL models?

Maybe we should have some granularity, like to download only the models we use for live video?

Reply:
Yeah I've been thinking we could have some kind of "tags" here, so you can download models only related to a specific pipeline or sth. I'll keep this as a separate improvement tho for the future, wdyt

Reply:
Yeah, let's keep it as a separate PR